common/recommend.js

```diff
@@ -1,12 +1,28 @@
 const https = require('./get');
 
+// courseID\">MATH\&? 152(?:(?:.|\n|\r)*?) Prerequisite((?:.|\n|\r)*?)<\/p>
+// courseID\">MATH\&? 152(.|\n|\r)*?<\/p>
 
 exports.getRecommended = (subject, classNumber, type, callback) => {
-    const regex = new RegExp ('courseID\\">' + subject.toUpperCase() + '(\\&?) ' + classNumber + '((.|\\n|\\r)*?)' + type + ': ((.|\\n|\\r)*?)<\\/p>','g'); 
+    // const regex = new RegExp ('courseID\\">' + subject.toUpperCase() + '\\&? ' + classNumber + '(?:(?:.|\\n|\\r)*?)' + type + ': ((?:.|\\n|\\r)*?)<\\/p>','g'); 
+    const regex = new RegExp ('courseID\\">' + subject.toUpperCase() + '\\&? '+  classNumber + '(.|\\n|\\r)*?<\\/p>','g'); 
+    const regex2 = new RegExp (type + ': (.*)?<\\/p>','g');
     let url = "https://www.bellevuecollege.edu/classes/All/" + subject.toUpperCase();
```

Comment:
This will be updated to use bellevuecollege.com/classes/all/MATH/070 for example soon